x/ibc/20-transfer/module.go

```diff
@@ -119,3 +124,123 @@ func (am AppModule) BeginBlock(ctx sdk.Context, req abci.RequestBeginBlock) {
 func (am AppModule) EndBlock(ctx sdk.Context, req abci.RequestEndBlock) []abci.ValidatorUpdate {
 	return []abci.ValidatorUpdate{}
 }
+
+// Implement IBCModule callbacks
+func (am AppModule) OnChanOpenInit(
+	ctx sdk.Context,
+	order channelexported.Order,
+	connectionHops []string,
+	portID string,
+	channelID string,
+	counterparty channeltypes.Counterparty,
+	version string,
+) error {
+	// TODO: Enforce ordering, currently relayers use ORDERED channels
+
+	if counterparty.PortID != types.PortID {
```

Comment:
Should we be checking this? Seems like it could be an issue if some other module claims "bank" portID on chainA and now a transfer module on chainA cannot connect to any other transfer module

Reply:
Not necessarily, it's not "safety-critical", just convention. I think it's useful convention for now though.